teerex/src/tests/tests.rs

```diff
@@ -632,38 +619,36 @@ fn verify_unshield_funds_from_enclave_not_bonding_account_fails() {
 }
 
 #[test]
-fn verify_call_confirmation_from_shards_not_enclave_fails() {
+fn confirm_confirm_processed_parentchainblock_works() {
```

Comment:
Are you double confirming? :D

Reply:
Haha, obviously :P `confirm_function_x_works` . But agreed, that's kinda unfortunate.  I will change that ;)

Reply:
haha, I honestly didn't get that. It's too late here, I guess. :P